README.md

```diff
@@ -23,16 +27,26 @@ Documentation is available for the extension and our APIs on the [Sailthru GetSt
 ##### Support
 For questions or troubleshooting, please visit [Sailthru Support][5] or submit an issue on GitHub.
 
-##### Migrating from previous versions
-The 2.0.0 uses updated schemas for User and Template Vars. Visit the documentation above to see the full schemas and what's changed.
+##### Migrating from 1.x
+* We've moved codepools. Before installing, make sure to remove the current plugin.
+* The 2.0.0 uses updated schemas for User and Template Vars. Visit the documentation above to see the full schemas and what's changed.
 
 ## Installation 
-* *We've moved codepools. Before installing, make sure to remove the current plugin.*
 * *After installation, you may need to login, clear cache, and then re-login.*
 
 ### Manual Installation
 
-#### With Modman
+#### (Migration-only) Removing the old extension
```

Comment:
Is removing the old extension referring to 1.x or 2.x branches in this instance?

Reply:
Removing 1.x (hence migration - goes with above section on migration from 1.x)